diff --git src/crypto/internal/boring/build-goboring.sh src/crypto/internal/boring/build-goboring.sh
index 4938b5eac3..4d2b935573 100755
--- src/crypto/internal/boring/build-goboring.sh
+++ src/crypto/internal/boring/build-goboring.sh
@@ -28,7 +28,7 @@ export CGO_ENABLED=0
 # Build and run test C++ program to make sure goboringcrypto.h matches openssl/*.h.
 # Also collect list of checked symbols in syms.txt
 set -e
-cd /boring/godriver
+cd ./boring/godriver
 cat >goboringcrypto.cc <<'EOF'
 #include <cassert>
 #include "goboringcrypto0.h"
