diff --git src/cmd/go/internal/vcweb/vcstest/vcstest_test.go src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
index 4a6d60039e..5749e06cc5 100644
--- src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
+++ src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
@@ -78,6 +78,7 @@ func serveStandalone(host string, port int) (err error) {
 // TestScripts verifies that the VCS setup scripts in cmd/go/testdata/vcstest
 // run successfully.
 func TestScripts(t *testing.T) {
+	t.Skip("Skipping due to ancient git version")
 	scriptDir, err := filepath.Abs(*dir)
 	if err != nil {
 		t.Fatal(err)
diff --git src/os/os_test.go src/os/os_test.go
index 94c3ad04f3..5fd6ac9310 100644
--- src/os/os_test.go
+++ src/os/os_test.go
@@ -1985,6 +1985,7 @@ func testWindowsHostname(t *testing.T, hostname string) {
 }
 
 func TestHostname(t *testing.T) {
+	t.Skip("Skipping due to incompatibility with libgethostname and NEWHOSTNAME")
 	t.Parallel()
 
 	hostname, err := Hostname()
