diff --git src/cmd/go/internal/vcweb/vcstest/vcstest_test.go src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
index 4a6d60039e..dfc3f8a922 100644
--- src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
+++ src/cmd/go/internal/vcweb/vcstest/vcstest_test.go
@@ -78,6 +78,8 @@ func serveStandalone(host string, port int) (err error) {
 // TestScripts verifies that the VCS setup scripts in cmd/go/testdata/vcstest
 // run successfully.
 func TestScripts(t *testing.T) {
+	t.Skip("Skipping due to ancient git version")
+
 	scriptDir, err := filepath.Abs(*dir)
 	if err != nil {
 		t.Fatal(err)
diff --git src/os/os_test.go src/os/os_test.go
index 277b2455e6..a2e38cf3f5 100644
--- src/os/os_test.go
+++ src/os/os_test.go
@@ -1749,6 +1749,8 @@ func testWindowsHostname(t *testing.T, hostname string) {
 }
 
 func TestHostname(t *testing.T) {
+	t.Skip("Skipping due to incompatibility with NEWHOSTNAME and libgethostname")
+
 	hostname, err := Hostname()
 	if err != nil {
 		t.Fatal(err)
